### PR TITLE
[FW][ADD][IMP]sale_dispatch_with_prepaid

### PR DESCRIPTION
**Pull request by fw-bot-adhoc**
For prepaid type's sales, allows dispatch even though the payment is not complete.

Forward-Port-Of: ingadhoc/sale#587

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/sale/pull/598).

**Comment by fw-bot-adhoc** [comment]
@jok-adhoc @matiasperalta1 this PR targets 16.0 and is the last of the forward-port chain.

To merge the full chain, say
> @fw-bot-adhoc r+

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port

**Comment by fw-bot-adhoc** [comment]
@jok-adhoc @matiasperalta1 ci/oba_without_tests failed on this forward-port PR

File: sale_dispatch_with_prepaid/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Dispatch with pre-paid',
+    'version': "15.0.1.1.0",
```
Comment (jok-adhoc) [inline sale_dispatch_with_prepaid/__manifest__.py:22]: cambiar version

**Comment by jok-adhoc** [comment]
r+

**Comment by jok-adhoc** [comment]
@roboadhoc r+

**Comment by roboadhoc** [comment]
@jok-adhoc you may want to rebuild or fix this PR as it has failed CI.

**Comment by roboadhoc** [comment]
I'm sorry, @jok-adhoc: I must know your email before you can review PRs. Please contact an administrator.

**Comment by jok-adhoc** [comment]
@roboadhoc r+ override=ci/oba_without_tests p=0

**Comment by roboadhoc** [comment]
@jok-adhoc you may want to rebuild or fix this PR as it has failed CI.

**Comment by roboadhoc** [comment]
I'm sorry, @jok-adhoc:
- I must know your email before you can review PRs. Please contact an administrator.
- you are not allowed to override this status.

**Comment by matiasperalta1** [comment]
@roboadhoc r+ override=ci/oba_without_tests p=0

**Comment by roboadhoc** [comment]
@matiasperalta1 you may want to rebuild or fix this PR as it has failed CI.

**Comment by fw-bot-adhoc** [comment]
@jok-adhoc @matiasperalta1 this PR was modified / updated and has become a normal PR. It should be merged the normal way (via @roboadhoc)

**Comment by matiasperalta1** [comment]
@roboadhoc r+

**Comment by roboadhoc** [comment]
@fw-bot-adhoc @matiasperalta1 because this PR has multiple commits, I need to know how to merge it:

* `merge` to merge directly, using the PR as merge commit message
* `rebase-merge` to rebase and merge, using the PR as merge commit message
* `rebase-ff` to rebase and fast-forward

**Comment by matiasperalta1** [comment]
@roboadhoc rebase-ff

**Comment by roboadhoc** [comment]
Merge method set to rebase and fast-forward.

**Comment by matiasperalta1** [comment]
@roboadhoc rebase-ff override=ci/oba_without_tests p=0

**Comment by roboadhoc** [comment]
Merge method set to rebase and fast-forward.

**Comment by jok-adhoc** [comment]
@nicomacr nico, buenas! Tenes el comando para mergear en este caso con 2 commits?

**Comment by nicomacr** [comment]
@jok-adhoc aca no lo mezcla por que hay un commit de mas, fijar de hacer un rebase para ajustar con la rama principal y luego squash para dejar un solo commit. asi lo podemos mezclar

cc @matiasperalta1

**Pull request closed without merging**